gobblin-data-management/src/main/java/gobblin/data/management/copy/extractor/FileAwareInputStreamExtractor.java

```diff
@@ -62,10 +62,8 @@ public FileAwareInputStream readRecord(@Deprecated FileAwareInputStream reuse)
       throws DataRecordException, IOException {
 
     if (!this.recordRead) {
-      Configuration conf = HadoopUtils.newConfiguration();
-      FileSystem fsFromFile = this.file.getOrigin().getPath().getFileSystem(conf);
       this.recordRead = true;
-      return new FileAwareInputStream(this.file, fsFromFile.open(this.file.getFileStatus().getPath()));
+      return new FileAwareInputStream(this.file, this.fs.open(this.file.getFileStatus().getPath()));
```

Comment:
This will break pull mode from multiple `FileSystem`s. Why is this change needed?